broadcast/src/buffered/mod.rs

```diff
@@ -0,0 +1,406 @@
+//! Cached, best-effort P2P Broadcast to all peers.
```

Comment:
nit: `Broadcast messages to and cache recent messages from untrusted peers.`

Reply:
Definitely don't need to use this exactly but I've been trying to use "doing" descriptions rather than "what" descriptions for these crates.